spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -443,7 +443,15 @@ definitions:
             desc: Microseconds portion of the time offset
         - flags:
             type: u8
-            desc: Status flags (reserved)
+            desc: Status flags
+            fields:
+              - 1-7:
+                  desc: Reserved, set to zero
+              - 0:
+                  desc: Weeks behavior
+                  values:
+                    - 0: Not affected on local timestamp rollover
+                    - 1: Incremented on local timestamp rollover
```

Comment:
Apologies if I am confused, but the description seems wrong? A value of 1 should have Starling ignore `MSG_GNSS_TIME_OFFSET` completely? In this case, we are sending both `SBP_MSG_PPS_TIME` and `MSG_GNSS_TIME_OFFSET`, can both co-exist without us ignoring one of them?

Reply:
If the flag is set to 1, Starling will ignore the message. But the description of the flag describes how the weeks field will work. Do you mean that we should explicitly state that the message will be ignored by Starling?

I’m not sure if they can co-exist without that flag being set, but that should not have changed with these changes. It will work the same way as before.

Reply:
On second thought, this could be independent from our PE and better describe the message itself.  Our PE can choose to either ignore the message or use it, but that would be independent from the description.